popup/popup.html

```diff
@@ -59,24 +59,27 @@ <h1>Typo Dashboard</h1>
                 <br />
                 <div class="flexcol">
                     <div class="flexrow flexrowMenu">
-                        <button type="button" id="imageagent">ImageAgent</button>
-                        <button type="button" id="markup">Markup</button>
-                        <button type="button" id="keybinds">Keybinds</button>
+                        <button type="button" id="translate">Translate (DE &rarr; EN)</button>
                     </div>
                     <div class="flexrow flexrowMenu">
-                        <button type="button" id="clearcanvas">Don't Clear</button>
+                        <button type="button" id="keybinds">Keybinds</button>
```

Comment:
Please provide a screenshot of the new popup layout.

Reply:
New popup  

![a](https://user-images.githubusercontent.com/12619603/153056346-ee5f2fc5-604f-4761-8885-cd0ad8ba867f.png)